### DISCUSSION TITLE
Preview first entry underlined

### DISCUSSION THREAD

**Discussion by lllllll20** [gokcehan/lf#1242] [Q&A] [answered]
Hi, I haven't used lf for a while, but now when i enable the preview mode, the first entry in the third column is underlined. Is there a way to stop this?

Thanks
![lf](https://github.com/gokcehan/lf/assets/63905394/dbda22b2-1a1f-4940-8054-d33f58c054e7)

**Comment by ilyagr** [comment, marked as answer]
Yes, to stop it you can do `set cursorpreviewfmt "\033[7m"`.

See the description of the option in https://pkg.go.dev/github.com/gokcehan/lf#hdr-Options for more details.

**Reply by lllllll20** [reply]
Thanks.
